src/applications/vaos/containers/AppointmentsPage.jsx

```diff
@@ -74,6 +76,7 @@ export class AppointmentsPage extends Component {
                 startNewAppointmentFlow={this.startNewAppointmentFlow}
               />
             )}
+            {showPastAppointments && <TabNav />}
```

Comment:
I moved the tab nav here so that we can maintain focus on the tabs when the page changes